control-plane-operator/controllers/hostedcontrolplane/oapi/params.go

```diff
@@ -126,7 +126,7 @@ func NewOpenShiftAPIServerParams(hcp *hyperv1.HostedControlPlane, globalConfig g
 			},
 		},
 		ReadinessProbes: config.ReadinessProbes{
-			oasContainerMain().Name: {
+			oauthContainerMain().Name: {
```

Comment:
changes in this file don't seem intended

Reply:
> changes in this file don't seem intended

Changes to this file fix existing bugs to enable readiness probe and resource request.

Reply:
Yup, those were the wrong container names, thx!